Library/docs/source/learn/users/management.md

```diff
@@ -0,0 +1,302 @@
+
+Keeping user information accurate and updated is important for a good user experience within CSP. Allowing users to update their display name provides flexibility and personalization. This feature helps users maintain current and relevant profiles, reflecting changes in their roles, preferences, or identities. A personalized display name ensures that users are easily recognizable within the platform, enhancing communication and collaboration.
+
+### Updating Display Name Method
+
+CSP provides a straightforward method for updating the display name: `UpdateUserDisplayName`. This method is part of the `UserSystem` and allows users to modify their display name while keeping other profile details intact.
+
+### Step-by-Step Guide to Updating Display Name
+
+Here's how to update a user's display name using the `UpdateUserDisplayName` method:
+
+**Step 1: Retrieving User Profile Information**
+
+Before making changes, you must retrieve the user's current profile information. This can be done using methods like `GetProfileByUserId` to ensure you have the correct user data.
+
+**Step 2: Modifying the Display Name**
+
+Once you have the user's profile, allow them to modify the display name to their desired value.
+
+**Step 3: Sending an Update Request**
+
+After modifying the display name, send the update request using the `UpdateUserDisplayName` method. The system will process the request and apply the changes.
+
+```
+void UserSystem::UpdateUserDisplayName(const String& UserId,
+                                       const String& NewDisplayName,
+                                       ProfileResultCallback Callback)
+{
+   auto Request = std::make_shared<UpdateDisplayNameRequest>();
+   Request->SetUserId(UserId);
+   Request->SetNewDisplayName(NewDisplayName);
+
+   const ResponseHandlerPtr ResponseHandler =
+      ProfileAPI->CreateHandler<ProfileResultCallback, ProfileResult,
+                                void, chs_user::ProfileDto>(Callback, nullptr,
+                                csp::web::EResponseCodes::ResponseOk);
+
+   static_cast<ProfileApi*>(ProfileAPI)->apiV1UsersDisplayNamePatch(Request, ResponseHandler);
+}
+```
+
+## User Profile Management Best Practices
+
+### Security Considerations
+
+To ensure user profile security in CSP, follow these practices:
+
+* **Use Strong Passwords**:  
+  * Encourage users to create strong passwords with a combination of uppercase and lowercase letters, numbers, and special characters.
+  * Passwords should be unique and changed regularly.
+
+* **Confirm Emails**:  
+  * Require users to confirm their email addresses during registration.
+  * Verified emails ensure accounts are legitimate and aid in password recovery.
+
+* **Avoid Plain Text Passwords**:  
+  * Always store passwords securely using encryption.  
+  * Never store passwords in plain text or send them via email.
+
+### User Experience Enhancements
+
+To enhance user experience in managing profiles, consider these strategies:
```

Comment:
[question] do you mean to say "to ensure *good* user experience in...."?

Reply:
Fixed in latest, thanks!